packages/beacon-node/src/node/notifier.ts

```diff
@@ -71,10 +71,12 @@ export async function runNodeNotifier(modules: NodeNotifierModules): Promise<voi
       // Notifier log lines must be kept at a reasonable max width otherwise it's very hard to read
       const tdProgress = chain.eth1.getTDProgress();
       if (tdProgress !== null && !tdProgress.ttdHit) {
-        tdTimeSeries.addPoint(tdProgress.tdDiffScaled, tdProgress.timestamp);
+        // TimeSeries accept time in Ms while timestamp is in Sec
+        tdTimeSeries.addPoint(tdProgress.tdDiffScaled, tdProgress.timestamp * 1000);
```

Comment:
Or TimeSeries.addPoint can take seconds instead and simplify the consumers since all want sec right?

Reply:
right! we can shift to seconds